Wheres-Waldo-Frontend/src/components/helpers/FetchCharacterInfo.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react";
+
+export default function FetchCharacterInfo(imageId) {
+	const [gameCharacters, setGameCharacters] = useState(null);
+	const [characterCoordinates, setCharacterCoordinates] = useState(null);
+
+	useEffect(() => {
+		fetch(`http://localhost:3000/image/${imageId}`)
```

Comment:
`localhost` is hardcoded here, so won't work when you deploy it somewhere. Should be able to use a relative url here instead